packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWModal/CWModal.tsx

```diff
@@ -41,13 +42,15 @@ const CWModal: FC<CWModalProps> = ({
   rootClassName,
   className,
   visibleOverflow,
+  zIndex,
 }) => (
   <ModalUnstyled
     open={open}
     onClose={onClose}
     slots={{ backdrop: Backdrop }}
     disableEnforceFocus
     className={rootClassName}
+    style={{ zIndex: zIndex || null }}
```

Comment:
lets do 
```
{...(zIndex >=0 && {style: {zIndex}})}
```

Reply:
> I'm not sure the why behind that change.

it was to avoid passing negative values (we already have issues dealing with positive ones). 

This code style here is not a big issue but I recommend we add predefined z-index global vars (previously discussed [here](https://github.com/hicommonwealth/commonwealth/pull/7166#discussion_r1528840681)) and only use those so these types of content overlay/cut-off bugs can be avoided.

Reply:
I agree we should cleanup z-indexes in the project, however, it should be done in separate PR (before we should figure out how exactly we want to tidy it up). For now, we need a fix to get rid of nasty bug